ubuntu/README.md

```diff
@@ -0,0 +1,673 @@
+  [8u45-8.7.0.5][175],
+  [8u51-8.8.0.3][176],
+  [8u60-8.9.0.4][177],
+  [8u65-8.10.0.1][178],
+  [8u66-8.11.0.1][179],
+  [8u72-8.13.0.5][180],
+  [8u92-8.15.0.1][181],
+  [8u102-8.17.0.3][182],
+  [8u112-8.19.0.1][183],
+  [8u121-8.20.0.5][184],
+  [8u131-8.21.0.1][185],
+  [8u144-8.23.0.3][186],
+  [8u152-8.25.0.1][187],
+  [8u162-8.27.0.7][188],
+  [8u172-8.30.0.1][189],
+  [8u181-8.31.0.1][190],
+  [8u192-8.33.0.1][191],
+  [8u202-8.36.0.1][192],
+  [8u212-8.38.0.13][193],
+  [8u222-8.40.0.25][194],
+  [8u232-8.42.0.21][195],
+  [8u232-8.42.0.23][196],
+  [8u242-8.44.0.11][197],
+  [8u252-8.46.0.19][198],
+  [8u262-8.48.0.51][199],
+  [8u272-8.50.0.21][200],
+  [8u275-8.50.0.53][201],
+  [8u282-8.52.0.23][202],
+  [8u302-8.56.0.21][203],
+  [8u312-8.58.0.13][204],
+  [8u322-8.60.0.21][205],
+  [8u332-8.62.0.19][206],
+  [8u342-8.64.0.15][207],
+  [8u345-8.64.0.19][208],
+  [8u352-8.66.0.15][209],
+  [8u362-8.68.0.19][210],
+  [8u362-8.68.0.21][211],
+  
+  * [7-latest][225],
+  [7u55-7.4.0.5][226],
+  [7u60-7.5.0.1][227],
+  [7u65-7.6.0.1][228],
+  [7u72-7.7.0.1][229],
+  [7u76-7.8.0.3][230],
+  [7u79-7.9.0.2][231],
+  [7u80-7.10.0.1][232],
+  [7u85-7.11.0.3][233],
+  [7u91-7.12.0.3][234],
+  [7u95-7.13.0.1][235],
+  [7u101-7.14.0.5][236],
+  [7u111-7.15.0.1][237],
+  [7u121-7.16.0.1][238],
+  [7u131-7.17.0.5][239],
+  [7u141-7.18.0.3][240],
+  [7u154-7.20.0.3][241],
+  [7u161-7.21.0.3][242],
+  [7u171-7.22.0.3][243],
+  [7u181-7.23.0.1][244],
+  [7u191-7.24.0.1][245],
+  [7u201-7.25.0.5][246],
+  [7u211-7.27.0.1][247],
+  [7u222-7.29.0.5][248],
+  [7u232-7.31.0.5][249],
+  [7u242-7.34.0.5][250],
+  [7u252-7.36.0.5][251],
+  [7u262-7.38.0.11][252],
+  [7u272-7.40.0.15][253],
+  [7u282-7.42.0.13][254],
+  [7u285-7.42.0.51][255],
+  [7u292-7.44.0.11][256],
+  [7u302-7.46.0.11][257],
+  [7u312-7.48.0.11][258],
+  [7u322-7.50.0.11][259],
+  [7u332-7.52.0.11][260],
+  [7u342-7.54.0.13][261],
+  [7u352-7.56.0.11][262],
+  
+  * [6-latest][263],
+  [6u49-6.4.0.6][264],
+  [6u53-6.5.0.2][265],
+  [6u56-6.6.0.1][266],
+  [6u59-6.7.0.2][267],
+  [6u63-6.8.0.1][268],
+  [6u69-6.9.0.3][269],
+  [6u73-6.10.0.3][270],
+  [6u77-6.11.0.2][271],
+  [6u79-6.12.0.2][272],
+  [6u83-6.13.0.3][273],
+  [6u87-6.14.0.1][274],
+  [6u89-6.15.0.1][275],
+  [6u93-6.16.0.1][276],
+  [6u97-6.17.0.1][277],
+  [6u99-6.18.0.3][278],
+  [6u103-6.19.0.1][279],
+  [6u107-6.20.0.1][280],
+  [6u113-6.21.0.3][281],
+  [6u119-6.22.0.3][282],
+  
+
+  [1]: https://www.azul.com/files/ZuluDocker60.gif
+  [2]: https://www.azul.com/
+  [3]: https://www.azul.com/products/core/
+  [4]: https://hub.docker.com/r/azul/zulu-openjdk-alpine
+  [5]: https://hub.docker.com/r/azul/zulu-openjdk-centos
+  [6]: https://hub.docker.com/r/azul/zulu-openjdk-debian
+  [7]: https://hub.docker.com/r/azul/zulu-openjdk
+  [8]: https://hub.docker.com/r/azul/zulu-openjdk/tags?name=distroless
+
+
+  [19]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/19-jre-headless-latest/Dockerfile
```

Comment:
This should point to https://github.com/zulu-openjdk/zulu-openjdk/

Reply:
Fixed